TCMSApp/src/client/app/layout/ht-top-nav.html

```diff
@@ -8,17 +8,17 @@
                 <span class="icon-bar"></span>
                 <span class="icon-bar"></span>
             </button>
-            <a class="navbar-brand" href="#">
+            <a class="navbar-brand" href="/">
                 <span class="glyphicon glyphicon-tasks" aria-hidden="true"></span>
             </a>
-            <a class="navbar-brand" href="#">
+            <a class="navbar-brand" href="/">
                 </span>{{vm.navline.text}}
```

Comment:
maybe something like somethingVm would be better. 
Why? So that we can access parent controller by its name from any child controller